src/Workspaces/Core/Portable/Shared/Extensions/IPropertySymbolExtensions.cs

```diff
@@ -36,15 +38,15 @@ public static IPropertySymbol RenameParameters(this IPropertySymbol property, IL
         }
 
         public static IPropertySymbol RemoveAttributeFromParameters(
-            this IPropertySymbol property, INamedTypeSymbol[] attributesToRemove)
+            this IPropertySymbol property, INamedTypeSymbol?[]? attributesToRemove)
         {
             if (attributesToRemove == null)
             {
                 return property;
             }
 
             bool shouldRemoveAttribute(AttributeData a) =>
-                attributesToRemove.Where(attr => attr != null).Any(attr => attr.Equals(a.AttributeClass));
+                attributesToRemove.Where(attr => attr != null).Any(attr => attr!.Equals(a.AttributeClass));
```

Comment:
We do have a WhereNotNull extension method that we could annotate ourselves if the ! is making you particularly sad here.

Reply:
(I weep though wondering why on earth why this helper handles nulls...)

Reply:
💭 Or I could use `attr?.Equals(...) ?? false`